packages/opencensus-core/src/trace/types.ts

```diff
@@ -28,10 +28,13 @@ export interface Tracing {
   /** Service to send collected traces to. */
   readonly exporter: exportersTypes.Exporter;
 
+  /** Gets active status  */
+  active: boolean;
+
   /**
-   * Enables the tracing process.
-   * @param userConfig A configuration object.
-   * @returns The tracing object.
+   * Starts the tracing.
```

Comment:
s/`tracing`/`Tracing instance`/g

Reply:
Ok. Updated here and in the implementation.